zillow_crawler/main.py

```diff
@@ -40,6 +44,7 @@ def main():
 
         for listing in listings:
             cons.print(listing.address)
+            total_houses += 1
 
         average_price = (
```

Comment:
I left this hair as a primer for creating some charts or tables.  Let me know if you are going to take that on or not.  If not, I can reevaluate and fill it in so that it functions correctly.

Reply:
I am not following :( if its related to potext, I cannot use it, I could not install it.  I have to comment it to run my code.

If its not that... than I am really not following :(

Reply:
What do you think about meeting this evening to clarify this?